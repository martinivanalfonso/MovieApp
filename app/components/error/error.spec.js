import React from 'react'
import { shallow } from 'enzyme'
import Error from './error'

describe('Error Component', () => {
  test('renders correctly', () => {
    const component = shallow(<Error />)
    expect(component).toMatchSnapshot()
  })
})
