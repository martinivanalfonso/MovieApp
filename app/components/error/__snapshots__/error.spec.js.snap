// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Component renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "black",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
        "fontSize": 18,
        "fontWeight": "bold",
      }
    }
  >
    Sorry, something went wrong!
  </Text>
</View>
`;
